src/main/java/org/dataone/portal/TokenGenerator.java

```diff
@@ -85,21 +91,18 @@ private TokenGenerator() throws IOException {
             @Override
             public void run() {
                 try {
-                    Certificate certificate = fetchServerCertificate();
-                    if (certificate != null) {
-                        RSAPublicKey currentKey = (RSAPublicKey) certificate.getPublicKey();
+                    Certificate NewServerCert = fetchServerCertificate();
```

Comment:
Any reason that the variable name starts with an uppercase letter?

Reply:
yes - this is because I'm an idiot. Fixed 601e7ff1a3d3f635db89b20de0495db4b1cc93de